### DISCUSSION TITLE
How can I keep italic, bold, H1 css styling from being written off?

### DISCUSSION THREAD

**Discussion by sterlingcouper** [bookfere/Ebook-Translator-Calibre-Plugin#80] [Q&A]
Hello,

Is there a way to make the plugin not write off the CSS styling in a book? I've tried it all but it keeps turning all italic and bold text into plain regular text.

There should be a clicking option that makes the plugin translate only the text while keeping everything else intact.

I love the plugin, thank you.

**Comment by wangshuihua** [comment]
We have considered your concern. Currently the plugin retains the CSS styling of the entire paragraph (translation unit) that is sent to the translation engine. However, it loses the styling of the text **within** each paragraph, such as the 'italic and bold text' that you noticed, which turns into plain regular text. 

In order to preserve the CSS styling of these "italic and bold text" **within** the translation paragraph, we would need to locate (identify) the styled text in the translated text received from engine. Due to the different sentence structures in various languages,  we haven't found a good way to accomplish this task.

**Reply by sterlingcouper** [reply]
As long as you've taken this into consideration for future updates that's fine by me. Thank you for your response.
